mlrun/feature_store/feature_vector.py

```diff
@@ -502,6 +502,11 @@ def get(self, entity_rows: List[Union[dict, list]], as_list=False):
             if not data:
                 data = None
             else:
+
+                requested_columns = [
```

Comment:
how is that related to mongo db?

Reply:
@guy1992l it is not related but it is a bug I fixed  - [fix bug when using "as" if the feature list]